android/app/src/main/java/me/rainbow/MainApplication.java

```diff
@@ -26,11 +26,11 @@
 import me.rainbow.NativeModules.RNTextAnimatorPackage.RNTextAnimatorPackage;
 import me.rainbow.NativeModules.RNZoomableButton.RNZoomableButtonPackage;
 import com.microsoft.codepush.react.CodePush;
+import com.facebook.react.config.ReactFeatureFlags;
 
 class RainbowJSIModulePackage extends ReanimatedJSIModulePackage {
     @Override
     public List<JSIModuleSpec> getJSIModules(ReactApplicationContext reactApplicationContext, JavaScriptContextHolder jsContext) {
-        MmkvModule.install(jsContext, reactApplicationContext.getFilesDir().getAbsolutePath() + "/mmkv");
```

Comment:
If you're removing this, you should remove [this import ](https://github.com/rainbow-me/rainbow/pull/3245/files#diff-f5b10a38d3df5e856664549745a6e4aa77e7e35a81a7e84b01559472870729c9R17)too